src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/SafeBuffer.cs

```diff
@@ -219,6 +226,11 @@ public void ReadArray<T>(ulong byteOffset, T[] array, int index, int count)
             ReadSpan(byteOffset, new Span<T>(array, index, count));
         }
 
+        /// <summary>
+        /// Reads value types from memory starting at the offset, and writes them into a span.</summary>
```

Comment:
I think we should say something about it using the length of the span to determine how much to read.